src/bindings/python/src/compatibility/ngraph/__init__.py

```diff
@@ -193,6 +193,13 @@
 from ngraph.opset11 import unsqueeze
 from ngraph.opset11 import variadic_split
 
+import warnings
+
+warnings.warn(
+    message="This API is deprecated and will be removed in 2024.0 release.",
```

Comment:
```suggestion
    message="OpenVINO API 1.0 is deprecated and will be removed in 2024.0 release.",
```
Is that also a message for C++ API? I think the message should actually point to what exact API is deprecated. @akuporos @ilyachur what do you think?

Reply:
I propose to allign the message with C++: https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/ie/ie_api.h#L56

Reply:
aligned